sys/net/link_layer/ieee802154/submac.c

```diff
@@ -27,8 +27,8 @@
 #define ENABLE_DEBUG 0
 #include "debug.h"
 
-#define CSMA_SENDER_BACKOFF_PERIOD_UNIT_US  (320U)
-#define ACK_TIMEOUT_US                      (864U)
+#define CSMA_SENDER_BACKOFF_PERIOD_UNIT_US   (320U)
+#define ACK_TIMEOUT_SYMS                     (54U)
```

Comment:
I was wondering where this value comes from and found an explanation [here](https://groups.google.com/g/omnetpp/c/q0hgsnVvJ0I).
Also helpful may be "6.3 PPDU format" (in 802.15.4-2006) to understand where the number of symbols comes from and the formula for `macAckWaitDuration`.
May it be that we would have to consider a different number of symbols for 868Mhz or is this neglectable for us?

Reply:
IIRC `macAckWaitDuration` also has to be calculated at runtime since some PHYs can have different values for the ACK timeout. But on 802.15.4-2020 there's no formula for the ACK timeout and can be chosen freely I think.

Reply:
Yea the standard is unnecessary cryptic here, you have to piece information together from multiple tables and have yourself asking if you got it right or not. I'm still not 100% sure the formulas used in the `at86rf215` driver are correct.

The most enlightening was the 'diff' contained in the IEEE 802.15.4-2012g ammendement:

![image](https://user-images.githubusercontent.com/1301112/216817645-1e8a5a68-6d8c-4408-a2ca-87b94e1accec.png)